src/Components/Keyboard/keyboard.js

```diff
@@ -1,51 +1,32 @@
 import "./keyboard.css";
 import React from "react";
 import { handleLetterClick } from "../../GameLogic/logic";
+import { __esModule } from "@testing-library/jest-dom/dist/matchers";
 
 export const Keyboard = () => {
-  const keys = [
-    "Q",
-    "W",
-    "E",
-    "R",
-    "T",
-    "Y",
-    "U",
-    "I",
-    "O",
-    "P",
-    "A",
-    "S",
-    "D",
-    "F",
-    "G",
-    "H",
-    "J",
-    "K",
-    "L",
-    "Z",
-    "X",
-    "C",
-    "V",
-    "B",
-    "N",
-    "M",
-    "Del",
-    "Enter",
-  ];
+  
+  const keyRows = ["QWERTYUIOP", "ASDFGHJKL", "ZXCVBNM"];
+  const backIcn = "🔙";
+  const enterTxt = "Enter";
```

Comment:
Since these two values are being used in your game logic, it would be better to store them as constants in a single location. Then we can reference the exact same value in the game logic and this component.